usaspending_api/search/v2/views/search.py

```diff
@@ -158,10 +158,45 @@ def post(self, request):
         if filters is None:
             raise InvalidParameterException("Missing one or more required request parameters: filters")
 
-        # filter queryset
-        queryset = transaction_filter(filters)
+        # build sql query filters
+        if can_use_view(filters, 'SummaryView'):
+            potential_scopes = ["agency", "subagency"]
+            if scope not in potential_scopes:
+                raise InvalidParameterException("scope does not have a valid value")
+            if scope == 'agency':
+                queryset = view_filter(filters, 'SummaryView')
+            else:
+                queryset = view_filter(filters, 'SummarySubagencyView')
+
+            if category == "awarding_agency":
+                queryset = queryset \
+                    .filter(awarding_agency_name__isnull=False) \
+                    .values(
+                        agency_name=F('awarding_agency_name'),
+                        agency_abbreviation=F('awarding_agency_abbr')) \
+                    .annotate(aggregated_amount=Sum('federal_action_obligation')) \
+                    .order_by('-aggregated_amount')
+            elif category == "funding_agency":
+                queryset = queryset \
+                    .filter(funding_agency_name__isnull=False) \
+                    .values(
+                        agency_name=F('funding_agency_name'),
+                        agency_abbreviation=F('funding_agency_abbr')) \
+                    .annotate(aggregated_amount=Sum('federal_action_obligation')) \
+                    .order_by('-aggregated_amount')
+
+            # Begin DB hits here
+            results = list(queryset[lower_limit:upper_limit + 1])
+            page_metadata = get_simple_pagination_metadata(len(results), limit, page)
+            results = results[:limit]
+
+            response = {"category": category, "scope": scope, "limit": limit, "results": results,
```

Comment:
This file also splits between using single quotes or double quotes for strings. Again, not required for this PR, just making a note.

Reply:
Which quotes do we prefer for this project?

Reply:
I think single quotes? So long as it's consistent.